cloudinventario

```diff
@@ -86,6 +91,27 @@ def collect(data):
      setproctitle.setproctitle(proctitle)
    return False
 
+# sentry load and apply config
+def sentryConfig(config, level):
+  dsn = (os.getenv("SENTRY_DSN") or config['sentry']['dns'])
+  environment= (os.getenv("SENTRY_ENVIRONMENT") or config['sentry']['env'])
+  traces_sample_rate= float(os.getenv("SENTRY_TSR") or config['sentry']['tsr'])
+  event_level= int(os.getenv("SENTRY_EVENT_LEVEL") or config['sentry']['event_level'])
```

Comment:
Init will fail if these optional config keys are not present, plz provide sensible default to make them really optional. Only `dsn` is required, if that is not present - do not init.

Reply:
In '4a7f19b' commit parameters are really optional.  Config if not found ENV return 'dev', TSR return '1.0' and EVENT_LEVEL return logging.ERROR.